mapApp/forms/hazard.py

```diff
@@ -27,11 +27,7 @@ class HazardForm(forms.ModelForm):
                 Field('date', id='hazard_date', template='mapApp/util/%s_datepicker.html', autocomplete='off'),
                 Field('hazard_category', id='hazard-category', type="hidden"),
                 Field('i_type', id='hazard-type'),
-            ),
-            AccordionGroup(
-                _('Description'),
-                Field('details', placeholder=_('optional')),
-                css_id='hazard-description',
+                Field('details', placeholder=_('mandatory')),
```

Comment:
Maybe consider 'required' instead of 'mandatory'?

Reply:
good catch! revised to "required" on all applicable forms.